shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integrated/test/http/combination/RateLimiterPluginTest.java

```diff
@@ -67,7 +67,7 @@ public void testSlidingWindow() throws IOException, ExecutionException, Interrup
         AdminResponse<Object> dto = rejectedRespFuture.get();
         assertEquals("You have been restricted, please try again later!", dto.getMessage());
 
-        Thread.sleep(1000);
+        Thread.sleep(5000);
```

Comment:
whats mean？

Reply:
@yu199195 Because replenishRate was set 0.2, we need to wait 5 seconds to be allowed.

Reply:
replenishRate was set 0.2 to reduce the probability of errors